dashboard/src/components/FunctionView/FunctionEditor.tsx

```diff
@@ -36,6 +36,9 @@ class FunctionEditor extends React.Component<IFunctionEditorProps> {
       return "javascript";
     } else if (runtime.match(/ruby/)) {
       return "ruby";
+    } else if (runtime.match(/php/)) {
+      // Doesn't work: Uncaught SyntaxError: Unexpected token <
```

Comment:
I receive here:
<img width="741" alt="screen shot 2018-04-13 at 18 13 37" src="https://user-images.githubusercontent.com/4025665/38745964-6fab3b1c-3f46-11e8-879f-85ef685ee9f3.png">

Reply:
I think it's because you're missing the import for php (see line 4)